src/makeCalculator.js

```diff
@@ -37,7 +37,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    DEFAULT_VALUE: 0,
+    result: 0,
+
+    operate(
+      modifyResult = N => N ?? this.result,
+      n = this.DEFAULT_VALUE,
+    ) {
+      // add context to the callback function and call it
+      modifyResult.bind(this)(n);
```

Comment:
We won't reuse this function. Why can't we just call it with our context. (let's use `call` instead of `bind`)